google-cloud-bigtable/lib/google/cloud/bigtable/instance.rb

```diff
@@ -744,6 +745,111 @@ def app_profiles
           AppProfile::List.from_grpc(grpc, service)
         end
 
+        # Gets the [Cloud IAM](https://cloud.google.com/iam/) access control
+        # policy for this instance.
+        #
+        # @see https://cloud.google.com/bigtable/docs/access-control
+        #
+        # @yield [policy] A block for updating the policy. The latest policy
+        #   will be read from the Bigtable service and passed to the block. After
+        #   the block completes, the modified policy will be written to the
+        #   service.
+        # @yieldparam [Policy] policy the current Cloud IAM Policy for this
+        #   instance
+        #
+        # @return [Policy] The current Cloud IAM Policy for this instance.
+        #
+        # @example
+        #   require "google/cloud/bigtable"
+        #
+        #   bigtable = Google::Cloud::Bigtable.new
+        #
+        #   instance = bigtable.instance("my-instance")
+        #   policy = instance.policy
+
+        def policy
+          ensure_service!
+          grpc = service.get_instance_policy(instance_id)
+          Policy.from_grpc(grpc)
```

Comment:
The docs say it will yield an object for updating, but I don’t see that happening here.

Reply:
Added a block yield to update policy